torchci/pages/metrics.tsx

```diff
@@ -101,8 +119,16 @@ function MasterCommitRedPanel({
   };
 
   return (
-    <Paper sx={{ p: 2, height: "100%" }} elevation={3}>
-      <ReactECharts
+    <Paper
+      sx={{
+        p: 2,
+        height: "100%",
+        backgroundColor: "var(--dropdown-bg)",
+        color: "var(--text-color)",
+      }}
+      elevation={3}
+    >
```

Comment:
It feels like we want a custom `ThemedPaper` component that adds background and foreground colors by default